packages/job-worker/src/playout/abPlayback/index.ts

```diff
@@ -30,11 +39,12 @@ export function applyAbPlaybackForTimeline(
 	abSessionHelper: AbSessionHelper,
 	blueprint: ReadonlyDeep<WrappedShowStyleBlueprint>,
 	showStyle: ReadonlyDeep<ProcessedShowStyleCompound>,
-	playlist: ReadonlyDeep<DBRundownPlaylist>,
+	playoutModel: PlayoutModel,
```

Comment:
I would like this to be changed back to `playlist`. 
It makes it a little simpler to reason about what the function might do, as then it can't by itself modify the `PlayoutModel`

Reply:
Only playlist are now parsed.